lib/indocker/artifacts/git.rb

```diff
@@ -1,13 +1,12 @@
 class Indocker::Artifacts::Git
-  attr_reader :name, :remote_name, :remote_url, :branch, :source_path, :target_path
+  attr_reader :name, :remote_name, :remote_url, :branch, :files
 
-  def initialize(name:, remote_name:, remote_url:, branch:, source_path:, target_path:)
+  def initialize(name:, remote_name:, remote_url:, branch:, files:)
```

Comment:
I would keep the previous way as well, and mark it as deprecated, to improve the migration process.

Reply:
Maybe we can even do not mark it as deprecated and use both solutions.

Reply:
👍